src/config_test.rs

```diff
@@ -180,25 +180,25 @@ mod get_table_key_value_from_config {
 
     #[test]
     fn handles_missing_table() {
-        let contents = "[hooks]";
+        let contents = r#"[hooks]"#;
         let result = get_table_key_value_from_config(contents, "foo", "");
         assert_eq!(result, Err(String::from("Missing config table")));
     }
 
     #[test]
     fn handles_missing_table_key() {
-        let contents = "[hooks]
+        let contents = r#"[hooks]
             pre-commit = 'cargo test'
```

Comment:
Now we can use standard double quotes within the string, instead of single quotes or having to escape the inner `"`!
```suggestion
            pre-commit = "cargo test"
```